src/database/migrations/20190818182539-create-user.js

```diff
@@ -3,6 +3,7 @@ export default {
     id: {
       allowNull: false,
       autoIncrement: true,
+      unique: true,
```

Comment:
hey @sebaztyn I believe this is an existing migration. We should try to not update an existing migration. A new migration should be made to make the changes you want.

Reply:
I have fixed this